src/Microsoft.Diagnostics.Monitoring.HostingStartup/Microsoft.Diagnostics.Monitoring.HostingStartup.csproj

```diff
@@ -9,7 +9,32 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <FrameworkReference Include="Microsoft.AspNetCore.App"/>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Tools\dotnet-monitor\ToolIdentifiers.cs" />
```

Comment:
Not something for this PR, but maybe it would be worthwhile to isolate commonly linked files.